tests/phpunit/Integration/JSONScript/docs/design.md

```diff
@@ -238,14 +238,13 @@ Sometimes certain data can cause inconsistencies with an environment hence it is
 <pre>
 {
 	"skip-on": {
-		"hhvm-*": "HHVM (or SQLite) shows opposite B1000, B9",
 		"mediawiki": [ ">1.30.x", "MediaWiki changed ..." ],
```

Comment:
Do we still need this test? We're skipping this test on all conditions.

Reply:
It's only an example on the documentation. It's not an actual test.